pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -30,6 +31,15 @@
 log.addHandler(logging.NullHandler())
 
 
+class Instrument(Instrument):
+
+    def check_errors(self):
+        errors = []
+        if self.read() != '':
+            errors.append(f'Incorrect return from previously set property')
+        return errors

```

Comment:
As you do not do any work with the lists, I'd do the logic differently:
```suggestion
        if self.read() == '':
            return []
        else:
            return ['Incorrect return from previously set property']
```